### PR TITLE
[v1  API Docs] some quick fixes

### PR DESCRIPTION
**Pull request by juliawu**
This PR:
* readds pagination section
* standardizes heading sizes
* standardizes "Info on" -> "Info of"
* fixes tag css on variable-group pages
* adds link to  bulk page on  v1/info/variable-groups

### CODE REVIEW

File: api/rest/v1/getting_started.md [hunk 1]
```diff
@@ -159,6 +159,54 @@ We've provided a trial API key for general public use. This key will let you try
 
 <b>The trial key is capped with a limited quota for requests.</b> If you are planning on using our APIs more rigorously (e.g. for personal or school projects, developing applications, etc.) please email us at [support@datacommons.org](mailto:support@datacommons.org?subject=API Key Request) with "API Key Request" in the subject to request an official key without any quota limits. We'll be happy to hear from you!
 
+### Pagination
```
[the remark this answers is not in the document]
Comment (juliawu) [reply api/rest/v1/getting_started.md:162]: I thought so too but it's missing in the current master branch...

File: api/rest/v1/info_variable_group.md [hunk 2]
```diff
@@ -45,7 +49,7 @@ https://api.datacommons.org/v1/info/variable-group/{VARIABLE_GROUP_DCID}?key={yo
...
 | ----------------------------------------------------------------- | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | key <br /> <required-tag>Required</required-tag>                  | string          | Your API key. See the [page on authentication](/api/rest/v1/getting_started#authentication) for a demo key, as well as instructions on how to get your own key. |
-| constrained_entities <br /> <optional-tag>Required</optional-tag> | Repeated string | Entities that the queried variable group's descendent variables have data for.                                                                                  |
+| constrained_entities <br /> <required-tag>Required</required-tag> | Repeated string | Entities that the queried variable group's descendent variables have data for.                                                                                  |
```
[the remark this answers is not in the document]
Comment (juliawu) [reply api/rest/v1/info_variable_group.md]: Converted to optional on both v1/ and  v1/bulk page, and changed the description.  What do you think?

**Review by shifucun — APPROVED** [review]
thanks for the update!

File: api/rest/v1/info_variable_group.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (shifucun) [reply api/rest/v1/info_variable_group.md]: Looking good!

**Pull request merged**